packages/flutter_tools/test/general.shard/args_test.dart

```diff
@@ -59,34 +58,45 @@ const String _needHelp = 'Every option must have help explaining what it does, e
 const String _header = ' Comment: ';
 
 void verifyOptions(String command, Iterable<Option> options) {
-  assert(command.contains('"'));
+  /*late*/ String target;
```

Comment:
This actually doesn't need to be late since it is definitely assigned (I believe it could even be final)